src/_pytest/python.py

```diff
@@ -143,9 +144,29 @@ def async_fail(nodeid: str) -> None:
     fail(msg, pytrace=False)
 
 
+def async_default_arg_warn(nodeid: str, function_name, param) -> None:
+    msg = (
```

Comment:
This would be much more readable with [f-strings](https://fstring.help/) rather than string concatenation.

Reply:
Sounds good, I'll change it to f-strings for more readability.